tests/components/derivative/test_sensor.py

```diff
@@ -186,7 +208,7 @@
 
     # Testing a power sensor at 1000 Watts for 1hour = 0kW/h
     assert round(float(state.state), config["sensor"]["round"]) == 0.0
-    assert state.attributes.get("unit_of_measurement") == "kW/h"
+    assert state.attributes.get("unit_of_measurement") == f"kW/{TIME_HOURS}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5054188)